File: FHLibrary/DomainModel/Passenger.cs
using System.ComponentModel.DataAnnotations;

namespace FHLibrary.DomainModel;
internal class Passenger
{
    //[DataType(DataType.EmailAddress)]
    //[EmailAddress]
    internal protected virtual string? email { get; set; }
    internal protected virtual string? password { get; set; }
    internal protected virtual string? passport { get; set; }
    internal protected virtual string? phone { get; set; }
    internal protected virtual DateTime? birth_date { get; set; }
    internal protected virtual string? nationality { get; set; }
    internal protected virtual string? first_name { get; set; }
    internal protected virtual string? last_name { get; set; }
    internal protected virtual string? addr_street { get; set; }
    internal protected virtual int? addr_stNo { get; set; }
    internal protected virtual IList<Feedback>? feedbacks { get; set; }
    internal protected virtual IList<Ticket>? tickets { get; set; }


    public Passenger()
    { 
        feedbacks = new List<Feedback>();
        tickets = new List<Ticket>();
    }
}

File: Client/js/Passenger.js
export class Passenger{
    constructor(email, password, passport, phone, birth_date, nationality, first_name, last_name, addr_street, addr_stNo, feedbacks, tickets){
        this.email=email;
        this.password=password;
        this.first_name=first_name;
        this.last_name=last_name;
        this.nationality=nationality;
        this.passport=passport;
        this.birth_date=birth_date;
        this.phone=phone;
        this.addr_street=addr_street;
        this.addr_stNo=addr_stNo;
        this.feedbacks=feedbacks;
        this.tickets=tickets;
    }
}